extensions/3rdparty/LyricWiki/Tag_Lyric.php

```diff
@@ -66,19 +65,10 @@
 	$wgHooks['AfterViewUpdates'][] = "efApplyIndexPolicy";
 }
 
-#Install extension
-function lyricTag()
-{
-	// Keep track of whether this is the first <lyric> tag on the page - this is to prevent too many Ringtones ad links.
-	global $wgFirstLyricTag;
-	$wgFirstLyricTag = true;
-}
-
 function lyricTag_InstallParser( $parser ) {
 	#install hook on the element <lyric>
 	$parser->setHook("lyric", "renderLyricTag");
 	$parser->setHook("lyrics", "renderLyricTag");
-	$parser->setHook("tonefuze", "renderToneFuzeTag");
```

Comment:
Are we sure that there are no more `<tonefuze>` tags in use? I guess it wouldn't be that bad if we found out after removing this code and seeing it inlined in some articles.